api/models/formation.go

```diff
@@ -118,6 +121,14 @@ func SetFormation(app, process string, count, memory int64) error {
 		params[fmt.Sprintf("%sMemory", UpperName(process))] = fmt.Sprintf("%d", memory)
 	}
 
+	if cpu > 0 {
+		if cpu > capacity.InstanceCPU {
+			return fmt.Errorf("requested cpu %d greater than instance size %d", cpu, capacity.InstanceCPU)
```

Comment:
Great validation. Though this makes me wonder: what happens if I set this, then scale my rack instance sizes down a bunch. I think we could do some nominal validation on the rack scale API too.